static/app/views/dashboards/manage/index.tsx

```diff
@@ -57,6 +68,14 @@ function ManageDashboards() {
   const navigate = useNavigate();
   const location = useLocation();
   const api = useApi();
+  const dashboardGridRef = useRef<HTMLDivElement>(null);
+
+  const [showTemplates, setShowTemplatesLocal] = useLocalStorageState(
+    SHOW_TEMPLATES_KEY,
+    shouldShowTemplates()
+  );
+  const [{rows, columns}, setGridSize] = useState({rows: 3, columns: 3});
```

Comment:
```suggestion
  const [{rows, columns}, setGridSize] = useState({rows: DEFAULT_NUM_ROWS, columns: DEFAULT_NUM_COLUMNS});
```

although, `DEFAULT_NUM_COLUMNS` doesn't actually exist. Maybe it should, and then you don't need `DEFAULT_NUM_WIDGETS`?